js/map.js

```diff
@@ -22,29 +22,22 @@ var map = svg.append( 'g' )
 // Define the div for the tooltip
 var div = d3.select("body")
     .append("div")
-    .attr("class", "tooltip")
-		.style("pointer-events", "none")
-		.style("background", "rgba(255,255,255,0.8)") //white, slightly transparent
-		.style("border", "3px")
-		.style("border-radius", "8px")
-		.style("font-family", "sans-serif");
+    .attr("class", "tooltip");
 		
 // setup var to map precip data to cells
 var precip = d3.map();
 
 // precip color scale
 var color = d3.scaleThreshold()
-    .domain(d3.range(0, 10)) // need better way to get range of data, should do inside queue
     .range(d3.schemeBlues[9]);
 
 // Add map features, need to queue to load more than one json
 d3.queue()
   .defer(d3.json, "../cache/state_map.geojson")
   .defer(d3.json, "../cache/county_map.geojson")
   .defer(d3.json, "../cache/precip_cells.geojson")
-  .defer(d3.csv, "../cache/precip_cell_data.csv", function(d) { 
-    precip.set(d.cell, +d.precip); 
-  })
+  .defer(d3.json, "../cache/precip_cell_data.json")
```

Comment:
did you get a sense of whether it was any faster to transfer and load as json vs csv?

Reply:
Oh, I did not. I used csv at first because that's what the target was caching already. I switched to json when I did my own processing step afterwards to be consistent with my other output formats.

Reply:
no worries. i think json files are probably larger (because they give the column name for every cell), but might be faster to load because the data types are known (numeric vs string). so i am curious, but we can figure it out another time.